Templates/JenkinsPipeline/Jenkinsfile

```diff
@@ -174,20 +176,29 @@ pipeline {
 							choice(
 								choices : ["build", "release", "preview"],
 								description: 'Please select the pipeline type',
-								defaultValue: 'build',
 								name: 'PipelineType'
 							),
 							choice(
-								choices : ["major", "minor", "patch"],
+								choices : ["patch", "minor", "major"],
 								description: 'Please provide the release type for automated computing of tags',
-								defaultValue: 'patch',
 								name: 'ReleaseType'
 							),
 							choice(
-								choices : ["no", "yes"],
+								choices : ["yes", "no"],
 								description: 'Enable pipeline logging',
-								defaultValue: 'no',
 								name: 'PipelineVerboseLogging'
+							),
+							choice(
+								choices : [ "merge","impact","file","metadata" ],
+								name : 'Lifecycle',
+								description: 'The zBuilder lifecycle type to run'
+							),
+							
+							string(
```

Comment:
I might have missed something, but I don't see the parameter being used in the script. In addition, this is a multi-branch pipeline. Users should not be required to select which branch to build.